easybuild/easyblocks/generic/bundle.py

```diff
@@ -199,6 +202,7 @@ def __init__(self, *args, **kwargs):
                 self.cfg.update('patches', comp_cfg['patches'])
 
             self.comp_cfgs.append(comp_cfg)
+            self.comp_blocks.append(comp_cfg.easyblock(comp_cfg))
```

Comment:
Since `self.comp_blocks` is not used anywhere and we are zipping both list in multiple steps, it would be more efficient to generate the zipped list here:
```suggestion
            self.comp_blocks.append((comp_cfg, comp_cfg.easyblock(comp_cfg)))
```
and then remove the `zip` calls in other steps.

Reply:
You're correct. When looking at `self.comp_cfgs`, there's also just a single use left. If we have both configs and blocks in a single list, we can change that single use as well and remove `self.comp_cfgs`.
 
I renamed the `comp_blocks` to `comp_instances` and changed the use of `comp_cfgs` in `check_checksums` to `comp_instances`.